lib/internal/Magento/Framework/Model/ResourceModel/Db/AbstractDb.php

```diff
@@ -383,6 +418,23 @@ protected function _getLoadSelect($field, $value, $object)
         return $select;
     }
 
+    /**
+     * Retrieve select object for load object data (locks record)
+     *
+     * @param string $field
+     * @param mixed $value
+     * @param \Magento\Framework\Model\AbstractModel $object
+     * @return \Magento\Framework\DB\Select
+     * @throws LocalizedException
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    protected function _getLoadSelectForUpdate($field, $value, $object)
```

Comment:
Implementation of `_getLoadSelect()` can be reused as follows:
```php
protected function _getLoadSelectForUpdate($field, $value, $object)
{
    return $this->_getLoadSelect($field, $value, $object)->forUpdate();
}
```

Reply:
Making that change now 👍